scripts/si_def_2_region_medium.json

```diff
@@ -29,7 +29,7 @@
     {
       "NamePrefix": "rkv-lab-si",
       "Region": "us-west-1",
-      "AvailabilityZone": "us-west-1a",
+      "AvailabilityZone": "us-west-1b",
```

Comment:
what is the reason to change az from 1a to 1b?

Reply:
there's some problem with provisioning vm in us-west-1a. "No Default VPC". Skipping it for now.